dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -313,6 +314,13 @@ private void deleteSelfFromEntries(User user) {
                 });
     }
 
+    // We don't delete the LambdaEvent because it is useful for other users
+    private void deleteSelfFromLambdaEvents(User user) {
+        lambdaEventDAO.findByUser(user).stream()
+                .filter(lambdaEvent -> lambdaEvent.getUser() == user)
```

Comment:
what's the difference between this filter and the DAO findByUser?

Reply:
D'oh, there isn't any. Will fix.